src/views/index.html

```diff
@@ -68,15 +68,48 @@ <h1 rows="2.0">
       <div></div>
       <div grid="rows" rows="1.5">
         <strong rows="0.5">
-          Applications for Hack Cambridge 2019 open soon.
+          {% if moment().isBefore(event.dates.getApplicationsStart()) %}
+            {% if moment().isBefore(event.dates.getApplicationsStart().subtract(1, 'weeks')) %}
+              Applications for Hack Cambridge 2019 open soon.
```

Comment:
Nice, this will mean we spend less time making this change each year!  We could go even further and replace "2019" with the event year.

Reply:
Good spot! I'll do that